src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -271,18 +271,79 @@ public static long DivRem(long a, long b, out long result)
             return div;
         }
 
-        internal static uint DivRem(uint a, uint b, out uint result)
+        [CLSCompliant(false)]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I think you can also mark these methods as `[NonVersionable]` so they get inlined accross versioning boundaries in R2R.